R30.s

```diff
@@ -281,7 +281,7 @@ V82GON		EXIT			# AVERAGE G ON.  USE CURRENT STATE VECTOR
 		2CADR	V82GON1
 
 		RELINT
-		CCS	NEWJOB		# WITHOLD V16 N44 UNTIL FIRST ORBIT CALC
+		CCS	NEWJOB		# WITHHOLD V16 N44 UNTIL FIRST ORBIT CALC
```

Comment:
`WITHOLD` is used in [Source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0519.jpg)